index.js

```diff
@@ -76,20 +76,21 @@ export function checkDataIntegrityProofFormat({
           for(const proof of proofs) {
             proof.should.have.property('type');
             proof.type.should.be.a(
-              'string', 'Expected `proof.type` to be a string.');
+              'string', 'Expected "proof.type" to be a string.');
           }
         });
-        it('"type" field MAY be either "DataIntegrityProof" or ' +
-          '"Ed25519Signature2020".', function() {
+        it('"type" field MUST be "DataIntegrityProof".', function() {
```

Comment:
might want to paramertize this to `"type" field MUST be "${expectedTypes.join(',')}` or something like that.